## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 511766`
Commit: `1ddff13b154ef88a2545212c7feaa5112e3457ab`

---
### `x86_64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>kdePackages.breeze-grub</li>
    <li>kdePackages.breeze-grub.debug</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>kdePackages.breeze-grub</li>
    <li>kdePackages.breeze-grub.debug</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>kdePackages.breeze-grub</summary>
<pre>@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;unpackPhase&quot; }
unpacking source archive /nix/store/00fhb6mz5r2n8l31vv8qspk72dda5dc1-breeze-grub-6.6.4.tar.xz
source root is breeze-grub-6.6.4
separate-debug-info: recording checksum of source files for debug support...
setting SOURCE_DATE_EPOCH to timestamp 1775548477 of file &quot;breeze-grub-6.6.4/mkfont.sh&quot;
Running phase: patchPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;patchPhase&quot; }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;updateAutotoolsGnuConfigScriptsPhase&quot; }
Running phase: configurePhase
@nix { &quot;action&quot;: &quot;setPhase&quot;, &quot;phase&quot;: &quot;configurePhase&quot; }
cmake flags: -GNinja -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/7sap8ffbxnrliy379qsdn5skngv5a64s-breeze-grub-6.6.4/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/7sap8ffbxnrliy379qsdn5skngv5a64s-breeze-grub-6.6.4/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/7sap8ffbxnrliy379qsdn5skngv5a64s-breeze-grub-6.6.4/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/7sap8ffbxnrliy379qsdn5skngv5a64s-breeze-grub-6.6.4/share/doc/breeze-grub -DCMAKE_INSTALL_INFODIR=/nix/store/7sap8ffbxnrliy379qsdn5skngv5a64s-breeze-grub-6.6.4/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/7sap8ffbxnrliy379qsdn5skngv5a64s-breeze-grub-6.6.4/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/7sap8ffbxnrliy379qsdn5skngv5a64s-breeze-grub-6.6.4/include -DCMAKE_INSTALL_SBINDIR=/nix/store/7sap8ffbxnrliy379qsdn5skngv5a64s-breeze-grub-6.6.4/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/7sap8ffbxnrliy379qsdn5skngv5a64s-breeze-grub-6.6.4/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/7sap8ffbxnrliy379qsdn5skngv5a64s-breeze-grub-6.6.4/lib -DCMAKE_STRIP=/nix/store/hb2bs5fg5wkm04x565737qd5nh2hy5nk-gcc-wrapper-15.2.0/bin/strip -DCMAKE_RANLIB=/nix/store/hb2bs5fg5wkm04x565737qd5nh2hy5nk-gcc-wrapper-15.2.0/bin/ranlib -DCMAKE_AR=/nix/store/hb2bs5fg5wkm04x565737qd5nh2hy5nk-gcc-wrapper-15.2.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/7sap8ffbxnrliy379qsdn5skngv5a64s-breeze-grub-6.6.4 -DQT_MAJOR_VERSION=6
CMake Warning:
  Ignoring extra path from command line:

   &quot;..&quot;


CMake Error: The source directory &quot;/build/breeze-grub-6.6.4&quot; does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.</pre>
</details>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>kdePackages.breeze-grub</summary>
<pre>copying path &#x27;/nix/store/z6731dsphsi98wzssaw7vhh30bf9ncdi-qtdeclarative-6.10.2&#x27; from &#x27;https://cache.nixos.org&#x27;...
copying path &#x27;/nix/store/2jaxs5mzn3x88gpih525iycxmaa41dc1-qmllint-validate-hook&#x27; from &#x27;https://cache.nixos.org&#x27;...
Running phase: qtPreHook
Running phase: unpackPhase
unpacking source archive /nix/store/00fhb6mz5r2n8l31vv8qspk72dda5dc1-breeze-grub-6.6.4.tar.xz
source root is breeze-grub-6.6.4
separate-debug-info: recording checksum of source files for debug support...
setting SOURCE_DATE_EPOCH to timestamp 1775548477 of file &quot;breeze-grub-6.6.4/mkfont.sh&quot;
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
cmake flags: -GNinja -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/7w4p32r0q242jshpqkayvmxvvrk7xcca-breeze-grub-6.6.4/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/7w4p32r0q242jshpqkayvmxvvrk7xcca-breeze-grub-6.6.4/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/7w4p32r0q242jshpqkayvmxvvrk7xcca-breeze-grub-6.6.4/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/7w4p32r0q242jshpqkayvmxvvrk7xcca-breeze-grub-6.6.4/share/doc/breeze-grub -DCMAKE_INSTALL_INFODIR=/nix/store/7w4p32r0q242jshpqkayvmxvvrk7xcca-breeze-grub-6.6.4/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/7w4p32r0q242jshpqkayvmxvvrk7xcca-breeze-grub-6.6.4/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/7w4p32r0q242jshpqkayvmxvvrk7xcca-breeze-grub-6.6.4/include -DCMAKE_INSTALL_SBINDIR=/nix/store/7w4p32r0q242jshpqkayvmxvvrk7xcca-breeze-grub-6.6.4/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/7w4p32r0q242jshpqkayvmxvvrk7xcca-breeze-grub-6.6.4/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/7w4p32r0q242jshpqkayvmxvvrk7xcca-breeze-grub-6.6.4/lib -DCMAKE_STRIP=/nix/store/waack2w5n0208w1qy7pxnwxbsq2n26vm-gcc-wrapper-15.2.0/bin/strip -DCMAKE_RANLIB=/nix/store/waack2w5n0208w1qy7pxnwxbsq2n26vm-gcc-wrapper-15.2.0/bin/ranlib -DCMAKE_AR=/nix/store/waack2w5n0208w1qy7pxnwxbsq2n26vm-gcc-wrapper-15.2.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/7w4p32r0q242jshpqkayvmxvvrk7xcca-breeze-grub-6.6.4 -DQT_MAJOR_VERSION=6
CMake Warning:
  Ignoring extra path from command line:

   &quot;..&quot;


CMake Error: The source directory &quot;/build/breeze-grub-6.6.4&quot; does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.</pre>
</details>
</details>

--- a/pkgs/applications/editors/neovim/tests/default.nix
+++ b/pkgs/applications/editors/neovim/tests/default.nix
@@ -91,7 +91,6 @@ let
     neovim-drv: buildCommand:
     runCommandLocal "test-${neovim-drv.name}"
       {
-        nativeBuildInputs = [ ];
         meta.platforms = neovim-drv.meta.platforms;
       }
       (

--- a/pkgs/build-support/rust/hooks/default.nix
+++ b/pkgs/build-support/rust/hooks/default.nix
@@ -75,7 +75,6 @@
 
   cargoSetupHook = makeSetupHook {
     name = "cargo-setup-hook.sh";
-    propagatedBuildInputs = [ ];
     substitutions = {
       defaultConfig = ../fetchcargo-default-config.toml;
 

--- a/pkgs/by-name/im/immer/package.nix
+++ b/pkgs/by-name/im/immer/package.nix
@@ -27,7 +27,6 @@ stdenv.mkDerivation (finalAttrs: {
 
   # immer is a header only library
   dontBuild = true;
-  buildInputs = [ ];
   dontUseCmakeBuildDir = true;
 
   doCheck = false;

--- a/pkgs/development/interpreters/lua-5/hooks/default.nix
+++ b/pkgs/development/interpreters/lua-5/hooks/default.nix
@@ -34,6 +34,5 @@ in
   # we move the files around ourselves
   luarocksMoveDataFolder = makeSetupHook {
     name = "luarocks-move-rock";
-    propagatedBuildInputs = [ ];
   } ./luarocks-move-data.sh;
 }

--- a/pkgs/development/interpreters/python/hooks/default.nix
+++ b/pkgs/development/interpreters/python/hooks/default.nix
@@ -40,7 +40,6 @@ in
     { makePythonHook }:
     makePythonHook {
       name = "conda-unpack-hook";
-      propagatedBuildInputs = [ ];
     } ./conda-unpack-hook.sh
   ) { };
 
@@ -48,7 +47,6 @@ in
     { makePythonHook }:
     makePythonHook {
       name = "egg-build-hook.sh";
-      propagatedBuildInputs = [ ];
     } ./egg-build-hook.sh
   ) { };
 
@@ -67,7 +65,6 @@ in
     { makePythonHook }:
     makePythonHook {
       name = "egg-unpack-hook.sh";
-      propagatedBuildInputs = [ ];
     } ./egg-unpack-hook.sh
   ) { };
 

--- a/pkgs/development/libraries/qt-5/modules/qtspeech.nix
+++ b/pkgs/development/libraries/qt-5/modules/qtspeech.nix
@@ -8,7 +8,6 @@
 
 qtModule {
   pname = "qtspeech";
-  propagatedBuildInputs = [ ];
   buildInputs = lib.optionals stdenv.hostPlatform.isLinux [ speechd-minimal ];
   nativeBuildInputs = [ pkg-config ];
   outputs = [

--- a/pkgs/development/ocaml-modules/camlgpc/default.nix
+++ b/pkgs/development/ocaml-modules/camlgpc/default.nix
@@ -24,9 +24,7 @@ buildDunePackage {
       hash = "sha256-znm+mX60RwYNCYXwm9HYCO8BRbzUM0Bm6dI1f1FzncA=";
     })
   ];
-  propagatedBuildInputs = [ ];
   doCheck = true;
-  checkInputs = [ ];
   meta = {
     description = "OCaml interface to Alan Murta's General Polygon Clipper";
     homepage = "https://github.com/johnwhitington/camlgpc";

--- a/pkgs/development/ocaml-modules/curl/default.nix
+++ b/pkgs/development/ocaml-modules/curl/default.nix
@@ -22,7 +22,6 @@ buildDunePackage (finalAttrs: {
     curl
   ];
 
-  checkInputs = [ ];
   doCheck = true;
 
   meta = {

--- a/pkgs/development/ocaml-modules/curl/lwt.nix
+++ b/pkgs/development/ocaml-modules/curl/lwt.nix
@@ -14,7 +14,6 @@ buildDunePackage (finalAttrs: {
     lwt
   ];
 
-  checkInputs = [ ];
   doCheck = true;
 
   meta = curl.meta // {

--- a/pkgs/development/ocaml-modules/dates_calc/default.nix
+++ b/pkgs/development/ocaml-modules/dates_calc/default.nix
@@ -20,8 +20,6 @@ buildDunePackage (finalAttrs: {
     sha256 = "sha256-B4li8vIK6AnPXJ1QSJ8rtr+JOcy4+h5sc1SH97U+Vgw=";
   };
 
-  propagatedBuildInputs = [ ];
-
   doCheck = true;
   checkInputs = [
     alcotest

--- a/pkgs/development/python-modules/simplenote/default.nix
+++ b/pkgs/development/python-modules/simplenote/default.nix
@@ -16,8 +16,6 @@ buildPythonPackage rec {
     sha256 = "1grvvgzdybhxjydalnsgh2aaz3f48idv5lqs48gr0cn7n18xwhd5";
   };
 
-  propagatedBuildInputs = [ ];
-
   meta = {
     description = "Python library for the simplenote.com web service";
     homepage = "http://readthedocs.org/docs/simplenotepy/en/latest/api.html";

--- a/pkgs/development/tools/misc/patchelf/unstable.nix
+++ b/pkgs/development/tools/misc/patchelf/unstable.nix
@@ -26,7 +26,6 @@ stdenv.mkDerivation {
   setupHook = [ ./setup-hook.sh ];
 
   nativeBuildInputs = [ autoreconfHook ];
-  buildInputs = [ ];
 
   doCheck = !stdenv.hostPlatform.isDarwin;
 

